packages/esm-patient-list-app/src/patient-list/patient-list-details.component.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { ExtensionSlot } from '@openmrs/esm-framework';
+import CustomOverflowMenu from '../ui-components/overflow-menu.component';
+import OverflowMenuVertical16 from '@carbon/icons-react/es/overflow-menu--vertical/16';
+import PatientListDataTable from '../patient-table/patient-table.component';
+import styles from './patient-list-detail.scss';
+
+const PatientListDetails: React.FC<{}> = () => {
```

Comment:
It should be `React.FC<PatientListDetailsProps>` where the `PatientListDetailsProps` are

```
interface PatientListDetailsProps extends RouteComponentProps {}
```

to reflect it coming from the React router. Also note that the params should ideally contain the `patientListUuid`